test/files/run/range.scala

```diff
@@ -55,6 +57,8 @@ object Test {
 
     val _grs = List[GR[_]](
       GR(BigDecimal(5.0)),
+      // SI-9348
```

Comment:
Please add the comment to the end of the next line, to avoid confusion that all of the subsequent cases are related to that ticket.